app/views/404.html

```diff
@@ -0,0 +1,31 @@
+<!-- Page that will show when user goes to a page that doesn't exist (404 error) -->
+<style>
+    #error-icon {
+        font-size: 6em;
+    }
+    #error-main {
+        font-size: 4em;
+        color: #0c629b;
+    }
+    #error-sub {
+        color: #627180;
+        font-size: 1.8em;
+    }
+    #error-desc {
+        font-size: 1em;
+    }
+    a.error-link {
+        color: #1074b9; 
+    }
+</style>
+
+<div style="text-align: center;">
```

Comment:
Let's add a margin for the `top`. Currently, the icon is not responsive and is going off screen.

![screenshot](https://user-images.githubusercontent.com/5673050/34093491-83fe6a8e-e3ee-11e7-97ea-ef686ff19d4a.png)